### PR TITLE
Nginx qa

### PR DESCRIPTION
**Pull request by TylerSheffels**
I think this resolves the remaining Nginx ssl issues we are seeing, but I want to go back and make sure.

### CODE REVIEW

File: ansible/roles/nginx/vars/cchq.yml [hunk 1]
```diff
@@ -3,23 +3,15 @@ nginx_sites:
...
-    try_files: "/dev/null @proxy_to_app"
-   location2:
-    name: "@proxy_to_app"
     use_balancer: True
-   location3:
-    name: "~* ^/(?:home|users|support|plans|images|pdfs|style|js|tour|div2|docs|fr|poc|es)/"
-    alias: "{{ public_site_path }}/$1"
-    try_files: "$uri $uri/index.html $uri/ =404"
-   location4:
+    proxy_redirect: "http://staging.commcarehq.org https://staging.commcarehq.org"
```
Comment (TylerSheffels) [inline ansible/roles/nginx/vars/cchq.yml]: Use variable

**Comment by TylerSheffels** [comment]
This should be ready for merge now. @benrudolph

**Comment by TylerSheffels** [comment]
@dannyroberts buddy ping

**Comment by czue** [comment]
whoa does that mean what i think it does?

**Comment by TylerSheffels** [comment]
It means that nginx is back to QA. I refuse to say anything more incriminating than that.

**Comment by benrudolph** [comment]
:dart:

**Pull request merged**